templates/pro/index.html

```diff
@@ -16,984 +24,633 @@
+     class="p-image-container__image"
+     width="852"
+     height="1279"
+     alt="" />',
+      "description_html": "<p>Canonical offers Ubuntu Pro at a discount for approved institutions, such as schools, research and academia.</p>",
+      "cta_html": "<a href='/contact-us/form?product=pro'>Contact us to learn more&nbsp;&rsaquo;</a>"
+      },
+      {
+      "title_text": "Personal use",
+      "image_html":  '<img src="https://assets.ubuntu.com/v1/a5677e3b-personal_use.png"
+     class="p-image-container__image"
+     width="852"
+     height="1279"
+     alt="" />',
+      "description_html": "<p>Ubuntu Pro is and always will be free for personal use on up to 5 physical machines, or 50 machines for <a href='https://wiki.ubuntu.com/Membership'>official Ubuntu Community members</a>.</p>",
+      "cta_html": "<a href='/pro/subscribe'>Get started today&nbsp;&rsaquo;</a>"
+      }
+      ]
+      ) %}
+    {% endcall %}
+
+    {% call(slot) vf_quote_wrapper(
+      is_shallow=true,
+      title_text="Used by professionals",
+      quote_size="small",
+      quote_text="Ubuntu Pro enables our engineering teams to focus on delivering industry-leading products and services to Acquia customers.<br /> <br />Canonical’s transparency and patching expedience give me peace of mind that we are providing the most secure and compelling solutions to power innovative digital experiences.",
+      citation_source_name_text="Robert Former",
+      citation_source_title_text="Chief Information Security Officer",
+      ) -%}
+
+      {%- if slot == 'signpost_image' -%}
+        <img src="https://assets.ubuntu.com/v1/ae29b16e-acquia_logo.png"
+             alt="Acquia" />
+      {%- endif -%}
+
+    {% endcall -%}
+    {% call(slot) vf_quote_wrapper(
+      quote_size="small",
+      quote_text="I manage my own compute cluster leveraging MAAS and other Canonical tools to support my research.<br /> <br />The open-source security patches delivered through Ubuntu Pro give my team peace of mind and ensure my servers are secure.<br /> <br />Canonical is continuously delivering timely CVE patches covering a broad portfolio of open-source applications for the entire ten-year lifetime of an Ubuntu LTS. This brings much needed stability and compliance.",
+      citation_source_name_text="David A Gutman",
+      citation_source_title_text="MD PhD",
+      citation_source_organisation_text="Associate Professor of Pathology"
+      ) -%}
+      {%- if slot == 'signpost_image' -%}
+        <img src="https://assets.ubuntu.com/v1/591b7338-emory_logo.png"
+             alt="Emory" />
+      {%- endif -%}
+
+    {% endcall -%}
+
+    {{ vf_divided_section(title={
+        "text": "Further reading",
+        },
+        is_split_on_medium=true,
+        blocks=[
+        {
+        "type": "divided-block",
+        "bullet_type": "none",
+        "items": [
+        {
+        "title_text": "The state of software supply chains research",
```

Comment:
Can probably ignore this comment, but in the designs these headings have less space below them.